src/vendor_upstream/dom/DOMMouseMoveTracker.js

```diff
@@ -52,34 +55,55 @@ class DOMMouseMoveTracker {
    */
   captureMouseMoves(/*object*/ event) {
     if (!this._eventMoveToken && !this._eventUpToken && !this._eventLeaveToken && !this._eventOutToken) {
-      this._eventMoveToken = EventListener.listen(
-        this._domNode,
-        'mousemove',
-        this._onMouseMove
-      );
-      this._eventUpToken = EventListener.listen(
-        this._domNode,
-        'mouseup',
-        this._onMouseUp
-      );
-      this._eventLeaveToken = EventListener.listen(
-        this._domNode,
-        'mouseleave',
-        this._onMouseEnd
-      );
-      this._eventOutToken = EventListener.listen(
-        this._domNode,
-        'mouseout',
-        this.onMouseEnd
-      );
+      if (!this._isTouchEnabled) {
```

Comment:
I don't think this is an either or scenario, since some devices have both a touch screen and a mouse (e.g. a surface).

Reply:
That's fair, didn't think about that. Changing to always binding mouse events and only bind touch events when touch is enabled.